ucm2/sof-soundwire/rt1316-1.conf

```diff
@@ -0,0 +1,22 @@
+# Use case Configuration for sof-soundwire card
+
+SectionDevice."Speaker" {
+	Comment	"Speaker"
+
+	EnableSequence [
+		cset "name='rt1316-1 DAC L Switch' 1"
+		cset "name='rt1316-1 DAC R Switch' 1"
+		cset "name='Speaker Switch' on"
+	]
+
+	DisableSequence [
+		cset "name='rt1316-1 DAC L Switch' 0"
+		cset "name='rt1316-1 DAC R Switch' 0"
+		cset "name='Speaker Switch' off"
+	]
+
+	Value {
+	      PlaybackPriority 100
+	      PlaybackPCM "hw:${CardId},2"
+	}
```

Comment:
No hw volume control?

Reply:
I'd like double check: The codec does not seem any volume control, but do we have any DSP (PGA) controls which can modify the stream volume to use them here?